plugin/index.php

```diff
@@ -21,7 +21,7 @@ function gutenberg_menu() {
 
 function gutenberg_scripts_and_styles( $hook ) {
 	if ( $hook === 'toplevel_page_gutenberg' ) {
-		wp_enqueue_style( 'gutenberg_css', plugins_url( 'style.css', __FILE__ ) );
```

Comment:
Curious if [`extract-text-webpack-plugin`](https://github.com/webpack-contrib/extract-text-webpack-plugin) (what we'd eventually want to use to pull out CSS) can be integrated easily into a development configuration.